node/clients/checkout.ts

```diff
@@ -46,17 +46,22 @@ export class Checkout extends JanusClient {
   public addItem = (
     orderFormId: string,
     items: Array<Omit<OrderFormItemInput, 'uniqueId' | 'index' | 'options'>>,
-    salesChannel?: string,
-    allowOutdatedData?: string[]
-  ) =>
+    options?: {
+      salesChannel?: string,
+      allowOutdatedData?: string[],
+      splitItem?: boolean 
+    }) =>
     this.patch<CheckoutOrderForm>(
       this.routes.addItem(
         orderFormId,
-        this.getChannelQueryString(salesChannel)
+        this.getChannelQueryString(options?.salesChannel)
       ),
-      {
+      { 
```

Comment:
Weird that the linter/formatter didn't catch this. Could you fix this and double-check that the linter is running properly for this file?
```suggestion
      {
```